iredis/entry.py

```diff
@@ -448,7 +448,7 @@ def create_client(params):
             scheme=dsn_from_url.scheme,
             username=dsn_from_url.username,
         )
-    if params["socket"]:
+    elif params["socket"]:
         client = Client(scheme="unix", path=params["socket"], db=db, password=password)
```

Comment:
```suggestion
        return Client(scheme="unix", path=params["socket"], db=db, password=password)
```

Let change this block if codes to this:


```
    if dsn_from_url:
        # db from command lint options should be high priority
        db = db if db else dsn_from_url.db
        return Client(
            host=dsn_from_url.host,
            port=dsn_from_url.port,
            db=db,
            password=dsn_from_url.password,
            path=dsn_from_url.path,
            scheme=dsn_from_url.scheme,
            username=dsn_from_url.username,
        )
    if params["socket"]:
        return Client(scheme="unix", path=params["socket"], db=db, password=password)
    return Client(host=host, port=port, db=db, password=password)
```

What do you think?

Reply:
That's a good idea! It's clearer than using `elif` . I fixed it.